src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -0,0 +1,280 @@
+      :disabled="showOrgPersonForm"  class="spacedButton" v-if="!hasRole(Roles.COMPLETING_PARTY, 1, 'ATLEAST')"
+      id="btn-add-cp">
+        <v-icon>mdi-account-plus-outline</v-icon>
+        <span>Add the Completing Party</span>
+      </v-btn>
+    </div>
+    <v-card flat class="people-roles-container" v-if="showOrgPersonForm">
+      <OrgPerson v-show="showOrgPersonForm"
+      :initialValue="currentOrgPerson"
+      :activeIndex="activeIndex"
+      :nextId="nextId"
+      :existingCompletingParty="completingParty"
+      @addEditPerson="onAddEditOrgPerson($event)"
+      @removePersonEvent="onRemovePerson($event)"
+      @resetEvent="resetData()"
+      @removeCompletingPartyRole="removeCompletingPartyAssignment()" />
+    </v-card>
+
+    <v-card flat v-if="orgPersonList.length > 0" :disabled="showOrgPersonForm" >
+      <ListPeopleAndRoles
+        v-if="orgPersonList.length > 0"
+        :personList="orgPersonList"
+        :isSummary="false"
+        @editPerson="editOrgPerson($event)"
+        @removePerson="onRemovePerson($event)"/>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Watch, Emit, Mixins } from 'vue-property-decorator'
+import { Action, State } from 'vuex-class'
+
+// Interfaces
+import { OrgPersonIF, BaseAddressObjIF, BaseAddressType, FormType, ActionBindingIF } from '@/interfaces'
+
+// Mixins
+import { EntityFilterMixin } from '@/mixins'
+
+// Enums
+import { EntityTypes, Roles, IncorporatorTypes } from '@/enums'
+
+// Components
+import OrgPerson from './OrgPerson.vue'
+import ListPeopleAndRoles from './ListPeopleAndRoles.vue'
+
+// Schemas
+import { addressSchema } from '@/schemas'
+
+@Component({
+  components: {
+    OrgPerson,
+    ListPeopleAndRoles
+  }
+})
+export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
+   $refs!: {
+    addPersonForm: FormType,
+    mailingAddressNew: BaseAddressType,
+    deliveryAddressNew: BaseAddressType
+  }
+
+   // State
+  @State(state => state.stateModel.addPeopleAndRoleStep.orgPeople)
+  readonly orgPersonList: OrgPersonIF[]
+
+  @Action setOrgPersonList!: ActionBindingIF
+  @Action setAddPeopleAndRoleStepValidity!: ActionBindingIF
+
+  private newOrgPerson: OrgPersonIF = {
+    id: null,
+    firstName: '',
+    lastName: '',
+    middleName: '',
+    orgName: '',
+    type: '',
+    roles: [],
+    address: {
+      mailingAddress: {
+        streetAddress: '',
+        streetAddressAdditional: '',
+        addressCity: '',
+        addressRegion: '',
+        postalCode: '',
+        addressCountry: '',
+        deliveryInstructions: ''
+      }
+    }
+  }
+
+  private showOrgPersonForm: boolean = false
+  private activeIndex: number = -1
+  private addEditInProgress: boolean = false
+  private currentOrgPerson: OrgPersonIF | null = null
+  private nextId: number = -1
+
+  readonly EntityTypes: {} = EntityTypes
+  readonly Roles: {} = Roles
+  readonly IncorporatorTypes: {} = IncorporatorTypes
```

Comment:
You don't need to declare types in a TS file (only in the old Vue class object).

Reply:
Done!